Fix text overflow in Source Interface replies

zishanmirza opened the pull request:
## Status

Ready for review

## Description of Changes

I couldn't find an issue for this. I hope it's alright if I fixed it. The issue is when long text is displayed in the "Read Replies" box, the text would overflow out of the box. This patch fixes it.

## Testing

1. make dev
2. Go to http://localhost:8080.
3. Click on "Log in".
4. Enter the test codename for last source (3/3) in the test file.
5. The long word no longer overflow out of the "Read replies" box with this patch.

## Deployment

N/A

## Checklist

### If you made changes to the server application code:

- [ ] Linting (`make lint`) and tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you added or removed a file deployed with the application:

- [ ] I have updated AppArmor rules to include the change

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

Choose one of the following:

- [ ] I have opened a PR in the [docs repo](https://github.com/freedomofpress/securedrop-docs) for these changes, or will do so later
- [ ] I would appreciate help with the documentation
- [X] These changes do not require documentation

legoktm commented on securedrop/sass/source.sass:
```diff
@@ -143,6 +143,7 @@ p#codename
     margin-bottom: 1em
     border-radius: 5px
     position: relative
+    word-wrap: break-word
```
According to <https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap>, the official name is "overflow-wrap" and "word-wrap" is an alias for legacy reasons, could you switch to overflow-wrap?

zishanmirza replied:
Sure, I can switch "word-wrap" to "overflow-wrap".

legoktm reviewed (changes requested):
Hi @zishanmirza, thanks for the PR! I tested it using your test plan and functionally it looks good to me.

Before:
![Screenshot 2022-02-22 at 16-11-43 SecureDrop Protecting Journalists and Sources](https://user-images.githubusercontent.com/81392/155241698-0fac38a9-d41c-4330-9c85-cd22e10b3b2b.png)

After:
![Screenshot 2022-02-22 at 16-12-32 SecureDrop Protecting Journalists and Sources](https://user-images.githubusercontent.com/81392/155241721-c5314ddd-935a-42ed-a264-0c935e5b6058.png)

Besides the one inline comment, I have one small nitpick: could you edit the first line of your commit message to be active and specify what component is being fixed. Like, "Fix text overflow in Source Interface replies".

Commit:
Fix text overflow in Source Interface replies

When long text is displayed in the "Read Replies" box, the text
would overflow out of the box. This patch fixes it.

Signed-off-by: Zishan Mirza <zishanm@pm.me>

Commit:
Fix text overflow in Source Interface replies

Switched "word-wrap" to "overflow-wrap".

Signed-off-by: Zishan Mirza <zishanm@pm.me>

ghost commented:
I have changed the commit message title.

legoktm reviewed (approved):
LGTM, thanks for the contribution!
